sentry_sdk/integrations/statsig.py

```diff
@@ -0,0 +1,39 @@
+from functools import wraps
+from typing import Any, TYPE_CHECKING
+
+import sentry_sdk
+from sentry_sdk.integrations import Integration, DidNotEnable
+
+import importlib
+
+try:
+    # The statsig package has the same name as this file. We use importlib to avoid conflicts.
```

Comment:
This wasn't an issue in the openfeature integration: https://github.com/getsentry/sentry-python/blob/9ff100a981e11c8f9bebd1ff51aee59864d693d4/sentry_sdk/integrations/openfeature.py#L6-L14